### PR TITLE
Add a rule for mark an account as paid

### PR DESCRIPTION
**Pull request by claudiaalves**

### CODE REVIEW

**Review by donbonifacio — APPROVED** [review]

File: src/clanhr/auth/authorization_rules.clj [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "Template role profiles for easy access"
   {:full-access ["admin" "hrmanager" "manager" "user" "" "staff"]
    :board-member-manager ["admin" "hrmanager" "manager" "staff"]
-   :board-member ["admin" "hrmanager" "staff"]})
+   :board-member ["admin" "hrmanager" "staff"]
```
Comment (lamuria) [inline src/clanhr/auth/authorization_rules.clj:8]: [non-related] @donbonifacio the `staff` was our old `developer` ?

**Review by lamuria — APPROVED** [review]

File: src/clanhr/auth/authorization_rules.clj (hunk 1 above)
Comment (donbonifacio) [reply src/clanhr/auth/authorization_rules.clj:8]: Hum... @lamuria I think the `staff` was something like support/product people, that could see hidden things.

Comment (claudiaalves) [reply src/clanhr/auth/authorization_rules.clj:8]: Should I add `staff` to the rule?

Comment (lamuria) [reply src/clanhr/auth/authorization_rules.clj:8]: humm we are not using it..

**Pull request merged**